src/org/zaproxy/zap/extension/pscanrulesAlpha/resources/Messages.properties

```diff
@@ -135,6 +135,13 @@ pscanalpha.serverheaderversioninfoleak.soln=Ensure that your web server, applica
 pscanalpha.serverheaderversioninfoleak.exploit=
 pscanalpha.serverheaderversioninfoleak.extrainfo=
 
+pscanalpha.serverheaderinfoleak.name=Server Leaks its Webserver Application via "Server" HTTP Response Header Field
+pscanalpha.serverheaderinfoleak.desc=The web/application server is leaking the application it uses as a webserver via the "Server" HTTP response header. Access to such information may facilitate attackers identifying other vulnerabilities your web/application server is subject to. This information alone, i.e. without a version string, is not very dangerous for the security of a server, nevertheless is this information in the Response Header Field almost always useless and thus just an obsolete attacking vector.
```

Comment:
I'm trying to say:

- The content of the `Server` header is almost always useless
- _Can_ be an attacking vectory